src/main/java/frc/robot/commands/TelescopeToPosition.java

```diff
@@ -12,6 +13,7 @@ public class TelescopeToPosition extends CommandBase {
 
     public TelescopeToPosition(TelescopeSubsystem telescopeSubsystem, double percent) {
         this.telescopeSubsystem = telescopeSubsystem;
+        percent = MathUtil.clamp(percent, telescopeSubsystem.LOWER_BOUND, telescopeSubsystem.UPPER_BOUND);
```

Comment:
why do we use a static constant here instead of an enum, but enums in other places?

Reply:
Here I'm not checking if it's in bounds, I just want it to stay in them. Those bounds are defined by these constants, so I believe it makes sense to use those constants here. However, this shows me a different issue: this subsystem takes in a percent. Clamping the value to 1000% would be disastrous. I am fixing that now.

Reply:
what i'm just wondering is why are the bounds stored in constants, and the bounds states are in enums?

Reply:
Why shouldn't they be? I've got 2 numbers and 3 states, it would be confusing to put them in the enum.

Reply:
we're going to have more numbers when we have presets, it might be nice to have all of them in the same place

Reply:
numbers that should be stored in the bounds enum? We'll have a separate enum for those, this is only for storing where the telescope is, tracking if it is under or over bounds, and behaving accordingly for manual control. we use those two constants  to find where we are on that scale.